drivers/gpio/gpio_davinci.c

```diff
@@ -64,10 +62,24 @@ struct gpio_davinci_config {
 	const struct pinctrl_dev_config *pcfg;
 };
 
+const unsigned int offset_array[5] = {0x10, 0x38, 0x60, 0x88, 0xb0};
```

Comment:
`struct gpio_davinci_regs` already has 4 bytes padding at the beginning. Those offsets here won't work.

Reply:
Oh, I see it was added last week. The tree I originally created the patches on was a bit older.